pkg/proto/configuration/blobstore/blobstore.proto

```diff
@@ -576,12 +576,14 @@ message LocalBlobAccessConfiguration {
     // Care should be taken that this value is not set too low. Every
     // epoch that still references valid data consumes 16 bytes of
     // memory and increases the size of the state file by a similar
-    // amount. This means that if this option is set to 5m, epoch
+    // amount. This means that if this option is set to "300s", epoch
     // bookkeeping consumes up to 12*24*365*16 B = ~1.68 MB of space if
     // the system were to operate for a full year without blocks being
-    // released. Setting this to 1s would blow this up by a factor 300.
+    // released. Setting this to "1s" blows this up by a factor 300.
     //
-    // Recommended value: 5m
+    // Recommended value: "300s"
+    // In JSON google.protobut.Duration is a string, time in seconds,
```

Comment:
My recommendation would remove this sentence, though, as we have many google.protobuf.Duration fields in our configuration and this isn't the most logical place to document it.

Maybe we should just link to https://developers.google.com/protocol-buffers/docs/proto3#json from README.md?

Reply:
That makes sense to me. Apologies for not checking the rest of the codebase for similar things.